app/renderer/components/navigation/navigationBar.js

```diff
@@ -128,6 +107,17 @@ class NavigationBar extends React.Component {
     props.bookmarkKey = siteUtil.getSiteKey(activeFrame)
     props.showHomeButton = !props.titleMode && getSetting(settings.SHOW_HOME_BUTTON)
 
+    props.location = location
+    props.isDarwin = isDarwin()
+    props.isFullScreen = isFullScreen()
+    props.bookmarkDetail = bookmarkDetail
+    props.menubarVisible = menuBarState.isMenuBarVisible(currentWindow)
+    props.siteSettings = state.get('siteSettings')
+    props.synopsis = state.getIn(['publisherInfo', 'synopsis']) || new Immutable.Map()
+    props.locationInfo = state.get('locationInfo')
+    props.titleMode = titleMode
+    props.isWideURLbarEnabled = getSetting(settings.WIDE_URL_BAR)
```

Comment:
@NejcZdovc Are the other props above necessary? Just asking for double check (as I am not sure at all about that).